File: seed/rust-sdk/simple-api/src/environment.rs
use serde::{Deserialize, Serialize};

#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
pub enum Environment {
    #[serde(rename = "production")]
    Production,
    #[serde(rename = "staging")]
    Staging,
}
impl Environment {
    pub fn url(&self) -> &'static str {
    match self {
        Self::Production => "https://api.example.com",
        Self::Staging => "https://staging-api.example.com",
    }
}
}
impl Default for Environment {
    fn default() -> Self {
    Self::Production
}
}

File: seed/ruby-sdk-v2/websocket-multi-url/lib/seed/environment.rb
# frozen_string_literal: true

module Seed
  class Environment
    PRODUCTION = { rest: "https://api.production.com", wss: "wss://ws.production.com" }.freeze
    STAGING = { rest: "https://api.staging.com", wss: "wss://ws.staging.com" }.freeze
  end
end
